extensions/amp-autocomplete/amp-autocomplete.md

```diff
@@ -192,7 +198,28 @@ Specifies the key to the data array within the JSON response. Nested keys can be
       </amp-autocomplete>
 
   </pre>
-      In the first example, the JSON payload is queued by the "items" key, and thus no component attribute is needed because the default value corresponds. In the second example, the JSON payload is queued by the "fruit" key, so the <code>items</code> attribute is given the value <code>"fruit"</code> so as to accurately etrieve the intended datasource. In both examples, the end user interaction is the same.
+[/filter] <!-- formats="websites" -->
+
+[filter formats="email"]
+
+  <pre lang="html">
+
+      <amp-autocomplete src="{{server_for_email}}/static/samples/json/amp-autocomplete-fruit-items.json">
+        <input type="text">
+      </amp-autocomplete>
+
+  </pre>
+  <pre lang="html">
+
+      <amp-autocomplete src="{{server_for_email}}/static/samples/json/amp-autocomplete-fruit-items.json" items="fruit">
+        <input type="text">
+      </amp-autocomplete>
+
+  </pre>
+
+[/filter] <!-- formats="email" -->
+
+In the first example, the JSON payload is queued by the "items" key, and thus no component attribute is needed because the default value corresponds. In the second example, the JSON payload is queued by the "fruit" key, so the <code>items</code> attribute is given the value <code>"fruit"</code> so as to accurately retrieve the intended datasource. In both examples, the end user interaction is the same.
```

Comment:
It can be unclear to readers what the JSON looks like because it's not included in the sample code. Should we include the JSON payload returned by the email endpoints separately?

Reply:
Good idea, included.